### PR TITLE
Xchain support: Fetch channel read/write entitlements and evaluate locally on the client

### PR DESCRIPTION
**Pull request by clemire**

### CODE REVIEW

File: packages/sdk/src/util.test.ts [hunk 1]
```diff
@@ -423,6 +428,15 @@ export async function expectUserCanJoin(
     wallet: ethers.Wallet,
 ) {
     const joinStart = Date.now()
+
+    // Check that the local evaluation of the user's entitlements for joining the space
+    // passes.
+    const entitledWallet = await spaceDapp.getEntitledWalletForJoiningSpace(spaceId, address, [
+        'http://127.0.0.1:8545',
+        'http://127.0.0.1:8546',
+    ])
```
Comment (texuf) [inline packages/sdk/src/util.test.ts:437]: are these our two anvils?

Comment (texuf) [reply packages/sdk/src/util.test.ts:437]: i see these are hard coded twice, should we be using getXchainSupportedRpcUrlsForTesting here?

**Review by texuf — APPROVED** [review]
one comment, looks good

File: packages/sdk/src/util.test.ts (hunk 1 above)
Comment (clemire) [reply packages/sdk/src/util.test.ts:437]: Yes, looks like I missed a spot - thank you!

**Pull request merged**